pkg/model/deployment.go

```diff
@@ -1,53 +1,63 @@
 package model
 
 import (
-	"os"
-
 	log "github.com/sirupsen/logrus"
 	appsv1 "k8s.io/api/apps/v1"
 	apiv1 "k8s.io/api/core/v1"
-	k8Yaml "k8s.io/apimachinery/pkg/util/yaml"
 )
 
 type deployment struct {
+	Name      string   `yaml:"name"`
 	File      string   `yaml:"file"`
 	Container string   `yaml:"container"`
 	Image     string   `yaml:"image"`
 	Command   []string `yaml:"command"`
 	Args      []string `yaml:"args"`
 }
 
+const (
+	// CNDRevisionAnnotation is the annotation added to a service to track the original deployment in k8
+	CNDRevisionAnnotation = "deployment.okteto.com/parent"
+
+	// CNDLabel is the label added to a dev deployment in k8
+	CNDLabel = "cnd"
```

Comment:
should we also use a full name here? That's what helm and kubernetes do for their labels.

Reply:
agree